exonum-java-binding/core/rust/exonum-java/TUTORIAL.md

```diff
@@ -56,34 +56,40 @@ service_name2 = "/path/to/service2_artifact.jar"
 
 ### Step 2. Generate Node Configuration
 
-EJB App configuration is pretty similar to configuration of any other Exonum service,
+Exonum Java app configuration is pretty similar to configuration of any other Exonum service,
 with a few additional parameters.
 
 #### Generate Template Config
 
 ```$sh
-$ ejb-app generate-template testnet/common.toml \
+$ exonum-java generate-template testnet/common.toml \
     --validators-count=1
 ```
 
 #### Generate Node Private and Public Configs
 
+**Note:** in this tutorial we are not providing any passwords for consensus and service private keys.
+Please consult Exonum documentation for the correct way of the node configuration.
```

Comment:
Does anybody have a link to a proper docs? I dunno

Reply:
actually, we waiting for a new documentation update :)

Reply:
"for the secure way …"?